src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,308 @@
+                        : '';
+
+                      return (
+                        <tr
+                          data-cy="person"
+                          key={people.slug}
+                          className={trClassName}
+                        >
+                          <td>
+                            <NavLink
+                              to={`${people.slug}`}
+                              className={classNames(
+                                {
+                                  'has-text-danger': people.sex === 'f',
+                                },
+                              )}
+                            >
+                              {people.name}
+                            </NavLink>
+                          </td>
+
+                          <td>{people.sex}</td>
+                          <td>{people.born}</td>
+                          <td>{people.died}</td>
+                          {necessaryPeople.find(mother => {
+                            return mother.name === people.motherName;
+                          }) ? (
+                              peoples.map(mother => {
+                                if (mother.name === people.motherName) {
+                                  return (
+                                    <td>
+                                      <NavLink
+                                        to={`${mother.slug}`}
+                                        className="has-text-danger"
+                                      >
+                                        {mother.name}
+                                      </NavLink>
+                                    </td>
+                                  );
+                                }
+
+                                return null;
+                              })
+                            ) : (
+                              <td>
+                                {
+                                  people.motherName ? people.motherName
+                                    : '-'
+                                }
+                              </td>
+                            )}
+
+                          {necessaryPeople.find(father => {
+                            return father.name === people.fatherName;
+                          }) ? (
+                              necessaryPeople.map(father => {
+                                if (father.name === people.fatherName) {
+                                  return (
+                                    <td>
+                                      <NavLink
+                                        to={`${father.slug}`}
+                                      >
+                                        {father.name}
+                                      </NavLink>
+                                    </td>
+                                  );
+                                }
+
+                                return null;
+                              })
```

Comment:
Better move all calculations out from JSX, if you can from code, to separate variables with meaningful variable naming